app/main.py

```diff
@@ -1 +1,22 @@
-# write your code here
+import os
+
+
+def move_file(command: str) -> None:
+    get_command = command.split(" ")
+    if get_command[0] == "mv" and len(get_command) == 3:
+        old_file_name = get_command[1]
+        new_file_name = get_command[2]
```

Comment:
You could unpack the `get_command` variable on the next line after obtaining it, this approach would improve your code readability by eliminating the need to obtain values via indexes on these lines.